thirdparty/runtime_environments_test.go

```diff
@@ -0,0 +1,20 @@
+package thirdparty
+
+import (
+	"context"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetImageNames(t *testing.T) {
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetImageNames")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	result, err := c.getImageNames(context.TODO())
```

Comment:
From my understanding, it's more in line with the rest of the Evergreen tests to use:
```
ctx, cancel := context.WithCancel(context.Background())
defer cancel()
```
rather than `context.TODO()`.